config/production.json

```
@@ -20,7 +20,7 @@
     "frontend": "https://frontend.opencollective.com",
     "website": "https://opencollective.com",
     "images": "https://images.opencollective.com",
-    "invoices": "https://invoices.opencollective.com"
+    "invoices": "https://pdf.opencollective.com"
```

If it's not disruptive (otherwise we can keep that for later) we should also rename the service itself in the config:

```suggestion
    "pdf": "https://pdf.opencollective.com"
```